src/frontends/pytorch/src/op/scatter.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/scatter_elements_update.hpp"
+#include "openvino/op/slice.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_scatter(const NodeContext& context) {
+    // Out-of-place schema
+    // "aten::scatter.src(Tensor self, int dim, Tensor index, Tensor src) -> Tensor"
+    // "aten::scatter.value(Tensor self, int dim, Tensor index, Scalar value) -> Tensor"
+
+    // Inplace schema
+    // "aten::scatter_.src(Tensor(a!) self, int dim, Tensor index, Tensor src) -> Tensor(a!)"
+    // "aten::scatter_.value(Tensor(a!) self, int dim, Tensor index, Scalar value) -> Tensor(a!)"
```

Comment:
it is not full list of schemas, there is any plans to support reduction?
 
 aten::scatter_.src(Tensor(a!) self, int dim, Tensor index, Tensor src) -> Tensor(a!):
  aten::scatter_.value(Tensor(a!) self, int dim, Tensor index, Scalar value) -> Tensor(a!):
  aten::scatter_.reduce(Tensor(a!) self, int dim, Tensor index, Tensor src, *, str reduce) -> Tensor(a!):
  aten::scatter_.value_reduce(Tensor(a!) self, int dim, Tensor index, Scalar value, *, str reduce) -> Tensor(a!):

Reply:
Yes, sorry, I've missed those schemas and though that all reductions are handled by separate aten::scatter_reduce op. I've already verified that reduce is working, just need to fix few cases that are still failing and do some cleanup in tests, it should be ready soon

Reply:
Added support for reduce schema, however, looking on all schemas there might still be few unsupported (attributes dimname, out) and some tests needed to be skipped due to issues with prim::Constant/prim::GetAttr

Reply:
dim name is rarely used and as I do not see it in any our models, so it is ok